### PR TITLE
Orchestra to csv

### PR DESCRIPTION
**Pull request by J4ken**

### CODE REVIEW

File: app/controllers/orchestra_controller.rb [hunk 1]
```diff
@@ -85,6 +85,15 @@ def show
       flash[:error] = t('orchestra.messages.show.failure')
       redirect_to '/'
     end
+    
+
+    @orchestraSignups = database.all_orchestra_signups params[:id]
+
+    respond_to do |format|
+      format.html
+      format.csv { send_data @orchestraSignups.encode('iso-8859-1'), :filename => 'orkestersammanställning.csv', :type => 'text/csv; charset=iso-8859-1; header=present' }
```
Comment (ljukas) [inline app/controllers/orchestra_controller.rb]: I think Date.now() should be addad to the file-name.

File: config/routes.rb [hunk 2]
```diff
@@ -25,6 +25,7 @@
   get 'orchestra/orchestras' => 'orchestra#orchestras'
   get 'orchestra/verify' => 'orchestra#verify_code'
   get 'orchestra/:id' => 'orchestra#show'
+  get 'orchestra/:id.csv' => 'orchestra#export'
```
Comment (patriksletmo) [inline config/routes.rb]: This route is not needed, all routes have the ending `(.:format)` added automatically. Please remove.

File: app/controllers/orchestra_controller.rb (hunk 1 above)
Comment (J4ken) [inline app/controllers/orchestra_controller.rb]: Done, fast det är Date.today ;D

File: app/controllers/orchestra_controller.rb [hunk 3]
```diff
@@ -86,6 +86,15 @@ def show
       flash[:error] = t('orchestra.messages.show.failure')
       redirect_to '/'
     end
+    
+
+    @orchestraSignups = database.all_orchestra_signups params[:id]
```
Comment (patriksletmo) [inline app/controllers/orchestra_controller.rb]: Don't fetch all orchestra signups unless the user requests the csv export.

File: app/controllers/orchestra_controller.rb [hunk 4]
```diff
@@ -86,6 +86,15 @@ def show
...
+
+    respond_to do |format|
+      format.html
+      format.csv { send_data @orchestraSignups.encode('iso-8859-1'), :filename => "orkestersammanställning-#{Date.today}.csv", :type => 'text/csv; charset=iso-8859-1; header=present' }
```
Comment (patriksletmo) [inline app/controllers/orchestra_controller.rb]: Do we want to capitalize the name? Should we include the current time as well? (These are only suggestions)

Comment (ljukas) [inline app/controllers/orchestra_controller.rb]: I dont think we want to capitalize, but should make it english, like "orchestrasummary-*date*". If you do Date.now() you get time aswell if I remember correctly.

Comment (patriksletmo) [inline app/controllers/orchestra_controller.rb]: I only get the date (without time) when I test this code myself.

**Comment by patriksletmo** [comment]
The database part of this pull request has been merged with master and this code can be tested by visiting https://sof-web-staging-pr-103.herokuapp.com/.

File: app/controllers/orchestra_controller.rb (hunk 4 above)
Comment (ljukas) [inline app/controllers/orchestra_controller.rb]: Thats because it says Date.today right now.

Comment (ljukas) [inline app/controllers/orchestra_controller.rb]: Alternativly one does it like Date.today + Time.now

Comment (ljukas) [inline app/controllers/orchestra_controller.rb]: Seems Im wrong, there is now Date.now. But the last thing i said with Time should work.

**Pull request merged**